fish_prompt.fish

```diff
@@ -1,3 +1,9 @@
+function fst; set_color -o fa0; end
+function snd; set_color -o 36f; end
+function trd; set_color -o f06; end
+function dim; set_color    666; end
+function off; set_color normal; end
```

Comment:
I would recommend moving each of these into a subfolder called `functions`, so for example `functions/fst.fish`, etc. This will guarantee that the whole thing works even if files are sourced in a different order, or functions are overridden (as was discovered with `fish_greeting`).

Reply:
I couldn't catch is there any naming convention in fish theme to avoid overriding global function. 

So I put `batman_color_` prefix on color name... just in case! If you don't like it, just tell me.

Reply:
the current implementation defines global functions with names that are pretty likely to conflict with aliases, so yeah, it makes sense to change the names.

i tend to use `__themename_*` for helper functions like this, because all functions do litter global namespace so it makes it less likely for someone to accidentally hit a helper via tab completion or something.

Reply:
fixed. Thanks for advice!

Reply:
now the function calls don’t match :)

fix those and we’ll get it merged!

Reply:
what a mistake! fixed.